cmd/performance-profile-creator/cmd/root.go

```diff
@@ -34,25 +32,23 @@ import (
 
 	performancev2 "github.com/openshift-kni/performance-addon-operators/api/v2"
 	machineconfigv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
-	clientmachineconfigv1 "github.com/openshift/machine-config-operator/pkg/generated/clientset/versioned/typed/machineconfiguration.openshift.io/v1"
 	v1 "k8s.io/api/core/v1"
-	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 const (
+	// ClusterScopedResources defines the cluster scope resources under
```

Comment:
`ClusterScopedResources defines the subpath, relative to the top-level must-gather directory[1] on which we find the cluster-scoped definitions saved by must-gather`
[1] please check